!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  control_frequency: 30.0
  Kp: 4.0
  Ki: 8.0
  Kd: 0.0
  desired_cube_x: 0.2
  desired_cube_y: 0.0
  desired_tag_x: 0.18
  desired_tag_y: 0.0
  seperate_I_threshold: 0.1
  max_velocity: 0.3
  min_velocity: 0.05
  max_angular_velocity: 0.3
  min_angular_velocity: 0.01
state: []
